product/urls.py

```diff
@@ -1,7 +1,7 @@
 from django.urls import path
-from django.urls.resolvers import URLPattern
-from .views      import ProductDetailView
+from .views      import ProductDetailView, ProductListView
 
 urlpatterns = [
+        path('', ProductListView.as_view()),
         path('/detail/<int:product_id>', ProductDetailView.as_view()),
```

Comment:
```suggestion
        path('/<int:product_id>', ProductDetailView.as_view()),
```
이미 id가 다음 뎁스 즉 개별적인 id값의 디테일을 설명하고 있으니 detail이라는 경로는 명시하지 않아도 될 것 같습니다.

Reply:
넵! 수정하였습니다~